content/References/IPv6.md

```diff
@@ -0,0 +1,248 @@
+IPv6 is not widely used, but ISPs worldwide are increasing their adoption of this protocol over IPv4 because it offers a significantly larger address space to support the ever growing Internet and the various devices connecting to networks through the Internet of things (IoT) and other emerging technologies.
+
+Existing IPv4 network infrastructure components might not be IPv6-compatible.
+This can result in costly changes to upgrade components or replace all or some of enterprise hardware.
+It takes time and effort to train IT professionals on how to implement and manage IPv6 networks.
+
+An IPv6 website or network cannot be seen by or communicated with by an IPv4 network without a gateway.
+IPv4 and IPv6 do not directly communicate with each other without the aid of either a gateway or other network configuration changes like dual-stacking the two protocols within the same network or deploying IPv6 tunneling to route traffic.
+
+IPv6 requires IP security where IPv4 security is optional. IPv6 uses an IPsec protocol to provide the required IP security.
+For more information, see [IPsec (Internet Protocol Security)](https://www.techtarget.com/searchsecurity/definition/IPsec-Internet-Protocol-Security).
+
+## Implementing IPv6
+Before implementing IPv6, there are a few things to consider:
+
+* {{< expand "Does your network infrastructure accommodate IPv6?" "V" >}}
+  
+  Most modern operating systems (Windows, Linux, MacOS) include IPv6 compatibility, but some Internet of Things (IoT) devices, older systems, and legacy network devices might require updates or replacement.
+
+  Organizations need to determine if the return on investment justifies the cost of converting to an all IPv6 infrastructure if your network components cannot accommodate v6 addresses.
+  Research your current infrastructure components to determine what is required to implement the network configuration that suits your use case.
+
+  Alternatively, you can migrate to IPv6 infrastructures through a staged approach. See [How enterprises can migrate from IPv4 to IPv6](https://www.techtarget.com/searchnetworking/tip/How-enterprises-can-migrate-from-IPv4-to-IPv6) for more information.
+{{< /expand >}}
+
+* {{< expand "Can your network infrastructure receive and send both IPv6 and IPv4 traffic?" "v" >}}
+
+  IPv6 and IPv4 networks cannot see or communicate with each other unless some [gateway](https://www.techtarget.com/iotagenda/definition/gateway) exists to allow for this.
+  Your DNS server needs to be capable of providing name resolution for these more complex IPv6 addresses.
+  
+  Specific network topology and configuration instructions and recommendations are outside the scope of this article.
+  Consult with your IT network administrator or service provider for how to equip your network to allow communication between both IP protocols.
+  
+  You can dual-stack IPv4 and IPv6 in SCALE if your system has either two network interface cards or a NIC with two ports.
```

Comment:
This is not correct.  Only one interface/NIC port is needed to dual-stack.  (Can just leave this sentence out.)

Reply:
You know I searched for this to remove references to dual stacking... I think I need new glasses or better typing skills to enter in the find window!